src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -78,25 +76,28 @@ def get_current_exposure(self, index: pd.Index) -> pd.Series:
 
 class StaticChildWasting(Risk):
     def __init__(self):
-        """
-        Parameters
-        ----------
-        risk :
-            the type and name of a risk, specified as "type.name". Type is singular.
-        """
-        self.risk = EntityString("risk_factor.child_wasting")
-        name = "static_child_wasting"
-        self.configuration_defaults = self._get_configuration_defaults()
-        self.exposure_distribution = self._get_exposure_distribution()
-        self._sub_components = [self.exposure_distribution]
+        # use super's init to get exposure distribution
+        # but overwrite other names
+        risk = "risk_factor.child_wasting"
+        super().__init__(risk)
```

Comment:
Nit: can do this in one line

Reply:
Oh yeah I was using risk on another line in a previous iteration but it's no longer necessary.